src/Auth0.OidcClient.Android/ChromeCustomTabsBrowser.cs

```diff
@@ -29,6 +29,13 @@ protected override void OpenBrowser(Android.Net.Uri uri, Context context = null)
             {
                 if (IsNewTask)
                     customTabsIntent.Intent.AddFlags(ActivityFlags.NewTask);
+
+                if (AutoCloseBrowser)
```

Comment:
I understand this is added to guarantee backwards compatibility, but I am curious to consider whether or not this is to be seen as a breaking change or a fix. Not saying this needs to be reverted, but curious to hear your thought as to why you went with the opt-in behavior?

Reply:
I did not want to assume everybody would want this change, so I provided a way to let the library work "as previously designed", to make sure an update to the library did not break anything for everyone else. Furthermore your code of conduct for any contribution states "no breaking changes" unless discussed with you...

Reply:
Assuming we only go with ClearTop, I am leaning towards thinking this is more a bug fix as it is a change that is to be considered breaking for the end user. I have a feeling the extra code here is making things more complicated while this probably could/should be the default behavior.

Reply:
If you prefer the auto-close functionality to be the default behavior, I can make that change if you want to, either by just forcing the extra flag(s) onto the intent (The least new code), or by defaulting the new property to "true" (the lease change to my suggested fix). What do you prefer?

Reply:
I will discuss this internally with the maintainer of the Android SDK, who's a bit more familiar with the different flags and whether or not it should have been the default or not. Will get back to this.